Add Thorsten Hans

ThorstenHans opened the pull request:
I've added myself to the list of speakers

ThorstenHans committed:
Add Thorsten Hans

Signed-off-by: GitHub <noreply@github.com>

karlhorky commented on README.md:
```diff
@@ -677,6 +677,14 @@ https://twitter.com/fhinkel
 
+### Saarbrücken
+
+<img src="https://avatars.githubusercontent.com/u/357884?v=4" height="70px" width="70px" align="left" alt="" />
```
we use the Twitter avatar in the avatars folder - so the way would be:

1. upload the image below to the avatars folder with the name `ThorstenHans` (matching your Twitter handle, no file extension)
2. set the `src` to `./avatars/ThorstenHans` (no file extension)

(I would also just do this for you, but I don't have push access to your branch - the checkbox on the right side of the screen in your PR that maintainers can make changes to your PR is not checked)

![ThorstenHans](https://github.com/karlhorky/awesome-speakers/assets/1935696/479744c3-2266-42fa-88a4-5251f791381e)

karlhorky reviewed (changes requested):
one small change mentioned above, then it's good to go

ThorstenHans committed:
chore: Update profile picture

As requested, this commit adds the Twitter profile picture to the avatars sub-folder and references it from their

Signed-off-by: GitHub <noreply@github.com>

karlhorky commented:
Thanks!
